packages/firestore/rollup.config.js

```diff
@@ -120,7 +103,14 @@ const es5Builds = [
       terser(terserOptions)
     ],
     external: id => deps.some(dep => id === dep || id.startsWith(`${dep}/`))
-  }
+  },
+  /**
+   * Browser CJS Build (based on the mangling in the ESM build above)
+   */
+  {
+    input: pkg.module,
+    output: { file: pkg.browser, format: 'cjs', sourcemap: true },
```

Comment:
The sourcemap generated by this step now maps to ESM file, rather than the actual sources. @Feiyang1 do you know how to fix this?

Reply:
Maybe something like this will help: https://webpack.js.org/loaders/source-map-loader/

Reply:
`rollup-plugin-sourcemaps` is what you need. :)
https://github.com/firebase/firebase-js-sdk/blob/master/packages/firebase/rollup.config.js#L64

I noticed that enums are not mangled and are still prefixed with `__PRIVATE_` in the built files. Can you skip enums when marking symbols to be mangled?

Reply:
Nice. That worked right away.

Re: Enums. In my original implementation, I tried to be smart and detect what the underlying types were as I parsed the code. This is really difficult to get right. I concluded that a type safe pipeline is too complex and fragile, and most of all does not buy as much in terms of size. Hence, I won't be able to tell at the callsite whether we are referring to an enum constant.

To solve the Enum trouble, I actually filed b/150211410 ("Reduce enum usage") this morning.

Reply:
Can you please add this dependency in `firestore/package.json` ?

Reply:
Done